pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,310 @@ excerpt:
+
+The public key associated with the custom CA certificate must be of type RSA.
+
+## ca\_certificate\_key\_path
+Path (realtive to the `$DCOS_INSTALL_DIR`) to a file containing the private key corresponding to the custom CA certificate, encoded in the OpenSSL (PKCS#8) PEM format. For example: `genconf/CA_cert.key`.
+
+**Note**: this is highly sensitive data. The configuration processor accesses this file only for configuration validation purposes, and does not copy the data. After successful configuration validation this file needs to be placed out-of-band into the file system of all DC/OS master nodes to the path `/var/lib/dcos/pki/tls/CA/private/custom_ca.key` before most DC/OS systemd units start up. The file must be readable by the root user, and should have have 0600 permissions set.
+
+Required if `ca_certificate_path` is specified.
+
+## ca\_certificate\_chain\_path
+Path (realtive to the `$DCOS_INSTALL_DIR`) to a file containing the complete CA certification chain required for end-entity certificate verification, in the OpenSSL PEM format. For example: `genconf/CA_cert_chain.pem`.
+
+The parameter must be left undefined if `ca_certificate_path` points to a root CA certificate.
+Required if `ca_certificate_path` is specified and if the custom CA certificate is an *intermediate CA certificate*. 
+
+For an intermediate CA, this needs to point to a file containing all CA certificates comprising the complete sequence starting precisely with the CA certificate that was used to sign the custom CA certificate and ending with a root CA certificate (where issuer and subject are equivalent), yielding a gapless certification path. The order is significant and the list must contain at least one certificate.
+
+
+# Installing DC/OS Enterprise with a custom CA certificate
+## Starting point
+Based on the requirements described above, this is the starting point for the installation:
+
+- The installation of DC/OS Enterprise via the Advanced Installer has been prepared according to the corresponding [documentation](/1.10/installing/ent/custom/advanced/). (up to the section **Install DC/OS** of that documentation)
+
+- On the bootstrap node, the files carrying custom CA certificate, the associated private key and, optionally, the CA certificate chain have been placed into the `$DCOS_INSTALL_DIR/genconf/` directory. (see the [section](#manually-placing-the-custom-ca-certificate-the-associated-private-key-and-the-certificate-chain-onto-the-bootstrap-node) above for more detailed description)
+
+- The custom CA private key has been securely placed on all DC/OS master nodes and is sufficiently protected (refer to [this section](#manually-placing-the-private-key-associated-with-the-custom-ca-certificate-onto-the-master-nodes) for more details). Example (command issued on one of the DC/OS master nodes):
```

Comment:
"custom CA private key" is a term that was not used in this document so far. Please use terms consistently.

Reply:
Please remove "and is sufficiently protected". It does not contain useful information.

Reply:
corrected